src/marks/link.js

```diff
@@ -2,7 +2,7 @@ import {geoPath, pathRound as path} from "d3";
 import {create} from "../context.js";
 import {curveAuto, PathCurve} from "../curve.js";
 import {Mark} from "../mark.js";
-import {coerceNumbers} from "../scales.js";
+import {coerceNumbers} from "../options.js";
```

Comment:
I moved the coercion routines up to options.js since coerceNumbers is now needed for valueof. I wanted to keep coerceDates together, but I left coerceSymbols where it was since that has more dependencies…